MachineLearning_Engine/MachineLearning_Engine.csproj

```diff
@@ -48,6 +48,10 @@
       <HintPath>..\..\Python_Toolkit\Build\Python_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Reflection_Engine, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Can you remove `Version`, `Culture` and `processorArchitecture`, and set `Copy Local` to false?